src/components/PaginationButtons/PaginationButtons.tsx

```diff
@@ -19,17 +20,17 @@ const PaginationButtons = ({ totalCount, activeNumberButton, numberPerPage, onCl
             const isActive = activeNumber === index + 1;
             const paginationButtonText = index + 1;
             return (
-              <li key={paginationButtonText} className='pagination-buttons__item'>
-                <button className={cx('pagination-buttons__button', isActive && 'pagination-buttons__button--active')} type="button" onClick={onClick} disabled={isActive}>{paginationButtonText}</button>
+              <li key={paginationButtonText} className='item'>
+                <button className={cx('button', isActive && 'button--active')} type="button" onClick={onClick} disabled={isActive}>{paginationButtonText}</button>
```

Comment:
should be styledButton without cx

Reply:
cx isn't used with styled

Reply:
Fixed: deleted cx, created styledPaginationButton